serialization/tests/TestSerializeTorchForce.cpp

```diff
@@ -34,18 +34,30 @@
 #include "openmm/internal/AssertionUtilities.h"
 #include "openmm/serialization/XmlSerializer.h"
 #include <iostream>
+#include <cstdio>
 #include <sstream>
+#include <torch/torch.h>
 
 using namespace TorchPlugin;
 using namespace OpenMM;
 using namespace std;
 
 extern "C" void registerTorchSerializationProxies();
 
-void testSerialization() {
-    // Create a Force.
+struct ExampleModuleImpl : torch::nn::Module { };
+TORCH_MODULE(ExampleModule);
+
+string writeExampleModuleFile() {
+    auto fileName = string(tmpnam(nullptr)) + ".pt";
+    ExampleModule module;
+    torch::save(module, fileName);
+    return fileName;
+}
```

Comment:
What are these changes for?  Why go to the work to build a new module and write it to disk, rather than just using `module.pt` which is already present?

Reply:
I modified these tests to try both the fileName and the jit Module constructors using an already available module